app/controllers/concerns/redux_state.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module ReduxState
```

Comment:
Thanks for doing this bro! Good to clean up the transformation definitions controller.